src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,17 +1,41 @@
 # TO-DO: complete the helpe function below to merge 2 sorted arrays
-def merge( arrA, arrB ):
-    elements = len( arrA ) + len( arrB )
+def merge(arrA, arrB):
+    elements = len(arrA) + len(arrB)
     merged_arr = [0] * elements
-    # TO-DO
-    
+    counter = 0
+    while len(arrA) > 0 and len(arrB) > 0:
```

Comment:
I like how you used a while loop here. The solution uses a for-in-range, but this works too!